aws-lambda-function/main.tf

```diff
@@ -38,3 +37,59 @@ resource aws_lambda_function lambda {
 
   tags = local.tags
 }
+
+
+resource aws_iam_role role {
+  name = local.name
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "lambda.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+
+  tags = local.tags
+}
+
+resource aws_cloudwatch_log_group log {
+  name = "/aws/lambda/${local.name}"
+}
+
+resource aws_iam_policy lambda_logging {
```

Comment:
I'm guessing that users of this module are going to want to supply customized roles or policies

Reply:
We usually use the inverse pattern - return the role name and let the caller attach things to it. I have an example coming shortly.